web_dashboard_tile/models/tile_tile.py

```diff
@@ -46,70 +84,113 @@ def _context_today():
+        'ir.model.fields',
+        string='Secondary Field',
+        domain="[('model_id', '=', model_id),"
+               " ('ttype', 'in', ['float', 'int'])]")
+    secondary_format = fields.Char(
+        string='Secondary Format',
+        help='Python Format String valid with str.format()')
+    secondary_value = fields.Char(
+        string='Secondary Value',
+        compute='_compute_data')
+    secondary_helper = fields.Char(
+        string='Secondary Helper',
+        compute='_compute_helper')
+
+    error = fields.Char(
+        string='Error Details',
+        compute='_compute_data')
+
     @api.one
     def _compute_data(self):
-        self.count = 0
-        self.computed_value = 0
-        if self.active:
-            # Compute count item
+        if not self.active:
+            return
+        try:
             model = self.env[self.model_id.model]
             eval_context = self._get_eval_context()
-            self.count = model.search_count(eval(self.domain, eval_context))
-            # Compute datas for field_id depending of field_function
-            if self.field_function and self.field_id and self.count != 0:
-                records = model.search(eval(self.domain, eval_context))
-                vals = [x[self.field_id.name] for x in records]
-                if self.field_function == 'min':
-                    self.computed_value = min(vals)
-                elif self.field_function == 'max':
-                    self.computed_value = max(vals)
-                elif self.field_function == 'sum':
-                    self.computed_value = sum(vals)
-                elif self.field_function == 'avg':
-                    self.computed_value = sum(vals) / len(vals)
-                elif self.field_function == 'median':
-                    self.computed_value = self.median(vals)
+            domain = self.domain or '[]'
+            count = model.search_count(eval(domain, eval_context))
+            if any([
+                self.primary_function and
+                self.primary_function != 'count',
+                self.secondary_function and
+                self.secondary_function != 'count'
+                    ]):
+                    records = model.search(eval(domain, eval_context))
+            for f in ['primary_', 'secondary_']:
+                value = 0
+                if self[f+'function'] == 'count':
+                    value = count
+                elif self[f+'function']:
+                    func = FIELD_FUNCTIONS[self[f+'function']]['func']
+                    if func and self[f+'field_id'] and count:
+                        vals = [x[self[f+'field_id'].name] for x in records]
+                        value = func(vals)
+                if self[f+'function']:
+                    self[f+'value'] = \
+                        (self[f+'format'] or '{:,}').format(value)
+                else:
+                    self[f+'value'] = False
+        except Exception as e:
```

Comment:
A large try/except like this with a catch for all exceptions is bad practice. You should try to minimize the size of your guards, as well as target specific exceptions that you are expecting.